api/ContactAddress.json

```
@@ -0,0 +1,53 @@
+{
+  "api": {
+    "ContactAddress": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": true
```

```suggestion
            "version_added": "84"
```

Based on https://storage.googleapis.com/chromium-find-releases-static/1a3.html#1a3c88d905e7b6079dd1d98ae39f1fad7a39f219